Src/WitsmlExplorer.Api/Workers/CreateRiskWorker.cs

```diff
@@ -0,0 +1,120 @@
+using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Serilog;
+using Witsml;
+using Witsml.Data;
+using Witsml.Extensions;
+using WitsmlExplorer.Api.Query;
+using Witsml.ServiceReference;
+using WitsmlExplorer.Api.Jobs;
+using WitsmlExplorer.Api.Models;
+using WitsmlExplorer.Api.Services;
+
+namespace WitsmlExplorer.Api.Workers
+{
+    public interface ICreateRiskWorker
+    {
+        Task<(WorkerResult, RefreshAction)> Execute(CreateRiskJob job);
+    }
+
+    public class CreateRiskWorker : ICreateRiskWorker
```

Comment:
Removing interface `CreateRiskWorker` and using `IWorker<CreateRiskJob>` will make the solution run at my end and everything worked as expected :+1 

```suggestion
    public class CreateRiskWorker : IWorker<CreateRiskJob>
```